### PR TITLE
[16.0][IMP-FIX] website_event_membership_restriction: Use variable to condition the button classes

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
Use variable to condition the button classes to avoid errors in the views and also adds better readability.

cc @Tecnativa

@pedrobaeza @chienandalu please review

### CODE REVIEW

File: website_event_membership_restriction/views/event_templates.xml [hunk 1]
```diff
@@ -17,9 +17,13 @@
             expr="//form//div[@id='o_wevent_tickets_collapse']//button"
             position="after"
         >
+            <t
+                t-set="is_disabled"
```
Comment (chienandalu) [inline website_event_membership_restriction/views/event_templates.xml]: Better not use such a generic name, as it could collide with other extensions.

Comment (pilarvargas-tecnativa) [reply website_event_membership_restriction/views/event_templates.xml]: changed

File: website_event_membership_restriction/views/event_templates.xml [hunk 2]
```diff
@@ -17,9 +17,13 @@
             expr="//form//div[@id='o_wevent_tickets_collapse']//button"
             position="after"
         >
+            <t
+                t-set="is_registration_disabled"
```
Comment (chienandalu) [inline website_event_membership_restriction/views/event_templates.xml]: This is still too generic... why not something like is_disabled_for_non_members

Comment (pilarvargas-tecnativa) [reply website_event_membership_restriction/views/event_templates.xml]: thanks

**Review by chienandalu — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**